CONTRIBUTING.md

```diff
@@ -27,3 +27,29 @@ for details, such as:
 * Development setup and testing your changes
 * Submitting a pull request and finding a reviewer
 
+To build and install the whole project from the source distribution,
+you need additional tools installed in your system including:
+
+* JDK 8 or later
+* Python 3
+* pip
+* setuptools
+* Go 1.12 or later
+* virtualenv
+* tox
+* Docker
+
+To install these in a Debian-based distribution:
```

Comment:
It would be great if you can specify debian release version you used, such as Debian 9 Stretch. The availability of package differ based on the version.

Reply:
I think it should work with any reasonably recent debian. I'm going to assume that until told otherwise.